app/services/HttpIdentityService.scala

```diff
@@ -59,14 +59,16 @@ class HttpIdentityService(apiUrl: String, apiClientToken: String)(implicit wsCli
 
   import IdentityServiceEnrichers._
 
-  private def request(path: String): WSRequest = {
+  private def request(path: String, headers: List[(String, String)], parameters: List[(String, String)]): WSRequest = {
     wsClient.url(s"$apiUrl/$path")
-      .withHttpHeaders("Authorization" -> s"Bearer $apiClientToken")
+      .withHttpHeaders(headers: _*)
```

Comment:
No need to pass in additional headers or parameters. You can just use the `withHeader()` and `withQueryStringParameters()` on the returned `WSRequest`.